mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLQueueDAO.java

```diff
@@ -23,11 +23,17 @@
 import com.netflix.conductor.core.execution.ApplicationException;
 import com.netflix.conductor.dao.QueueDAO;
 
-public class MySQLQueueDAO extends MySQLBaseDAO implements QueueDAO {
+public class MySQLQueueDAO extends MySQLBaseDAO implements QueueDAO, Closeable {
+    private static final Long UNACK_SCHEDULE_MS = 60_000L;
+    private final ScheduledExecutorService unackScheduler;
```

Comment:
This is the reference I was suggesting should be made static or moved off into a separate queue sweeper, I will push something to my branch to show what I meant.

Reply:
Good point and the @singleton annotation should solve our issue. If you want to move the processAllUnacks, please do. That would be a good move